dev9/Dev9/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace Dev9
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string[] lines = new string[2];
+            string path = @"c:\Users\source.txt";
+            try
+            {
+                lines = File.ReadAllLines(path);
+                if (lines[0] == String.Empty || lines[1] == String.Empty)
+                {
+                    Console.WriteLine("Incorrect file data format");
+                    Console.ReadLine();
+                    return;
+                }
+            }
+            catch
+            {
+                Console.WriteLine("Incorrect file data format.");
+                Console.ReadLine();
+                return;
+            }
+            
+            char[] lineOneInChar = new char[lines[0].Length];
+            char[] lineTwoInChar = new char[lines[1].Length];
+            lineOneInChar = lines[0].ToCharArray();
```

Comment:
What for? you make too many unnecessary convertions from char to string and back, not effective